task_reactredux/task1/src/Constants.js

```diff
@@ -16,3 +16,8 @@ export const errorMessagesEnum = Object.freeze({
     PasswordErrorMessage: "Пароль минимум 6 символов"
 });
 
+export const ON_MAIL_CHANGE = 'ON_MAIL_CHANGE';
+export const ON_PASSWORD_CHANGE = 'ON_PASSWORD_CHANGE'
+export const VALIDATE_MAIL = 'VALIDATE_MAIL';
+export const VALIDATE_PASSWORD = 'VALIDATE_PASSWORD';
```

Comment:
Please remove unused constants. Also, it's good practice to move action type constants to separate file in actions folder `../src/actions/types.js`